OamLog.yaml

```diff
@@ -341,6 +382,7 @@ paths:
                         - subscriber-application
                         - subscriber-release-number
                         - subscriber-operation
+                        - subscriber-protocol
                         - subscriber-address
                         - subscriber-port
                       properties:
```

Comment:
Similar to above comment, line 393:: PromptForBequeathingDataCausesRObeingRequestedToNotifyWithdrawnApprovalsToNewRelease:: subscriber-application :: description ::
Can we update the description from path of http-server/application-name to "http-client/application-name of NewRelease" since the new-release application-name will be updated in NewRelease LTP instance.

Because, if an application is having a new name other than http-server/application-name, the subscription needs to be made to new Release's application-name instead of current server application's name .

Reply:
Changed as proposed.

Reply:
Thanks for updating the path.

Also, the first line of description shall be changed to "Future application name"

Reply:
Changed as proposed.